_includes/content/ssh-add-keys-to-agent.md

```diff
@@ -0,0 +1,92 @@
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#mac-add-keys" data-group="mac">Mac</a></li>
+  <li><a data-toggle="tab" data-target="#win-add-keys" data-group="win">Windows</a></li>
+  <li><a data-toggle="tab" data-target="#linux-add-keys" data-group="linux">Linux</a></li>
+</ul>
+<div class="tab-content">
+<div id="mac-add-keys" class="tab-pane fade in active">
```

Comment:
Tested this, but it didn't work as reliably as the current strategy:
```
{% capture some-label %}
 MARKDOWN CONTENT 
{% endcapture %}
{{ some-label | markdownify }}
```
 So leaving as is for now.